### PR TITLE
Adds react-native-inappbrowser-reborn as peer to Bare RN OAuth

### PR DESCRIPTION
**Pull request by Ariflo**
### 📦 Pull Request

Adds `react-native-inappbrowser-reborn` as a peer dependency to our Bare RN OAuth Extension. 

### ✅ Fixed Issues

n/a

### 🚨 Test instructions

n/a

### ⚠️ Don't forget to add a [semver](https://semver.org/) label!

- `patch`: Bug Fix?
- `minor`: New Feature?
- `major`: Breaking Change?
- `skip-release`: It's unnecessary to publish this change.

### CODE REVIEW

File: packages/@magic-ext/react-native-bare-oauth/package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "@types/crypto-js": "~3.1.47"
   },
   "peerDependencies": {
-    "@magic-sdk/react-native-bare": ">=13.0.0"
+    "@magic-sdk/react-native-bare": ">=13.0.0",
+    "react-native-inappbrowser-reborn": ">=3.7.0"
```
Comment (Ethella) [inline packages/@magic-ext/react-native-bare-oauth/package.json:39]: shall we move this to devDependencies, as it's already peer?

https://github.com/magiclabs/magic-js/pull/432/files#diff-00b1eb415b6404cff50cddee27041bbc8742807d50be7d5ced8916f78236dda2L31

**Review by Ethella — APPROVED** [review]

File: packages/@magic-ext/react-native-bare-oauth/package.json (hunk 1 above)
Comment (Ariflo) [reply packages/@magic-ext/react-native-bare-oauth/package.json:39]: Good catch!

**Review by Ethella — APPROVED** [review]
LGTM. Do we need to update the graphCMS as well?

**Comment by Ariflo** [comment]
> LGTM. Do we need to update the graphCMS as well?

Yep, [already done](https://magic.link/docs/auth/login-methods/social-logins/integration/oauth-implementation/react-native) :)

**Comment by magiclabsFE** [comment]
:rocket: PR was released in [`@magic-ext/react-native-bare-oauth@7.1.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/react-native-bare-oauth%407.1.1), [`@magic-ext/react-native-expo-oauth@7.1.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/react-native-expo-oauth%407.1.1), [`@magic-sdk/react-native-bare@13.1.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/react-native-bare%4013.1.1), [`@magic-sdk/react-native-expo@13.1.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/react-native-expo%4013.1.1) :rocket:

**Pull request merged**